rest_auth_toolkit/serializers.py

```diff
@@ -24,6 +26,23 @@ class Meta:
             'password': {'style': {'input_type': 'password'}},
         }
 
+    def validate(self, data):
+        user = User(**data)
+
+        password = data.get('password')
```

Comment:
`dict.get` is useful for an optional field, but here `password` is required.  Direct item access is cleaner:

```python
password = data['password']
```

Reply:
Done!